File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/Southwest.java

package restaurant.sa.com.sarestaurant.appview.restaurant.model;

import com.google.gson.annotations.Expose;

public class Southwest {

    @Expose
    private Double lat;
    @Expose
    private Double lng;

    public Double getLat() {
        return lat;
    }

    public void setLat(Double lat) {
        this.lat = lat;
    }

    public Double getLng() {
        return lng;
    }

    public void setLng(Double lng) {
        this.lng = lng;
    }

}

File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/Southwest.kt
package restaurant.sa.com.sarestaurant.appview.restaurant.model
import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import com.google.gson.annotations.Expose

@Entity(tableName = "Southwest")
data class Southwest(
        @ColumnInfo(name = "latitude1") var lat: Double?,
        @ColumnInfo(name = "longitude2") var lng: Double?)